docs/get-started/yarn-gpu.md

```diff
@@ -117,3 +117,39 @@ On all masters:
 ```bash
 sudo systemctl restart hadoop-yarn-resourcemanager.service
 ```
+
+Note: If `cgroup` is mounted on `tmpfs` and a node is rebooted,
```

Comment:
I'm fine with adding a note if our users ran into this.  Is there any existing cgroup or YARN documentation we can point to instead?

Reply:
should this doc work?
https://manpages.ubuntu.com/manpages/bionic/man7/cgroups.7.html
![image](https://user-images.githubusercontent.com/84758614/189597921-71993aa3-e1e6-4b06-9a0d-96ff8c9c9b69.png)

Reply:
unfortunately that doesn't explicitly talk about reboot so I'm fine with leave this text here if it helps users.